data/pages.yml

```diff
@@ -232,3 +232,9 @@
       url: "adding-new-features"
     - title: "Repositories"
       url: "repositories"
+
+- title: "Glossary"
```

Comment:
One thought is to split the glossary into different "categories" (each a file), which adds a problem instead of fixing the current one.
But it's a good thought.